### PR TITLE
Added : Home page

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (49)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+body{
+    color:rgb(41, 41, 41);
```
Comment (PuneetP16) [inline index.css]: use variables instead of direct values

**Review by PuneetP16 — APPROVED** [review]

**Comment by PuneetP16** [comment]
Screenshot is not uploaded, can you provide live preview link?

**Review by PuneetP16 — APPROVED** [review]

File: index.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    "desc image";
+}
+.navbar{
+   position: relative;
+}
+.content-description{
+    grid-area: desc;
+    font-size: 2.5rem;
+    font-weight: bold;
+    padding: 2rem 1rem;
```
Comment (manojsarna) [inline index.css]: can use font weight values, 500, 600 etc

File: index.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    background-color: var(--purple-color);
+    color: var(--white);
+    position: relative;
+    width: 100%;
+}
+ @media (max-width:480px){
+    body{
+        font-size: 63.5%;
+    }
+    .navbar{
```
Comment (manojsarna) [inline index.css]: 63.5 or 62.5!!

do check your calculations

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                </div>
+                <div class="content-button">
+                    <a href="/src/components/Installation/Install.html"><button class="btn btn-purple btn-small">Get Started  </button></a>
+                    <a href="https://github.com/aiman-ansari/AiraveUI"><button class="btn btn-outline-purple btn-small">Github</button></a>
+                </div>
+            </div>
+                <img src="/src/assest/Source code-amico.png" class="image"/>
+        </div>   
+        <footer>
+            Made with <i class="fa fa-heart pd--5 text-secondary"></i> by Aiman Ansari
```
Comment (manojsarna) [inline index.html]: use Beautify to align your code

**Review by manojsarna — CHANGES_REQUESTED** [review]
Overall looks good!!

Do Check Comments

**Review by sanu29 — APPROVED** [review]
Neat and clean code looks good to merge

**Pull request merged**